src/matrix/matrixnormal.jl

```diff
@@ -109,7 +109,8 @@ end
 
 function logkernel(d::MatrixNormal, X::AbstractMatrix)
     A  = X - d.M
```

Comment:
In cases where `A'` does play nicely with `\`, this creates an additional copy, could there be a solution that avoids that?

Reply:
The main issue is [here](https://github.com/JuliaStats/PDMats.jl/blob/7ce4a5c609b288ab4b0ebaf3c397c9b46f7ce6a2/src/pdiagmat.jl#L45), for instance. I don't think anyone even remembers why that had to be `StridedVecOrMat`, and it's probably just going to be gotten rid of: [here](https://github.com/JuliaStats/PDMats.jl/pull/89), and [here](https://github.com/JuliaStats/PDMats.jl/pull/90). But for now it's why `S \ A'` doesn't work in general. So I can submit a PR to `PDMats`, wait for merge/release, and then come back here and kill this extra copy. Or else keep it for now, and submit another PR later whenever `PDMats` is updated.

Reply:
I'm ok to continue as is, can you post an issue here as a reminder of what has to be done once the change is done in PDMats?

Reply:
Definitely.